src/app/components/scrollBar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+const ScrollBarContainer = styled.div({
+  overflowY: 'hidden',
+  scrollbarGutter: 'stable',
+  ':hover': {
+    overflowY: 'auto',
+  },
+  '&::-webkit-scrollbar': {
+    width: 8,
+    height: 10,
+    background: 'transparent',
+  },
+  '&::-webkit-scrollbar-thumb': {
+    width: 8,
+    maxHeight: 10,
+    borderRadius: 24,
+    background: 'var(--white-800, rgba(255, 255, 255, 0.20))',
```

Comment:
is --white-800 defined anywhere?

why don't we use the props.theme.colors.white_800 and props.theme.colors.white_600?